src/components/MovieDetails/MovieDetails.tsx

```diff
@@ -0,0 +1,56 @@
+import './movie-details.css';
+
+export interface MovieDetailsProps {
+  imageUrl: string;
+  movieName: string;
+  releaseYear: number;
+  rating: number;
+  duration: number;
+  description: string;
+}
+
+const MovieDetails = ({
+  imageUrl,
+  movieName,
+  releaseYear,
+  rating,
+  duration,
+  description,
+}: MovieDetailsProps) => {
+  const defaultDuration = '0h';
```

Comment:
Read this  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters